pkg/util/logging/textlogger.go

```diff
@@ -0,0 +1,15 @@
+package logging
+
+import (
+	"os"
+
+	"k8s.io/klog/v2/textlogger"
+)
+
+var (
+	// TextLoggerConfig is the default configuration for the text logger.
+	TextLoggerConfig = textlogger.NewConfig(
+		textlogger.Verbosity(4),
+		textlogger.Output(os.Stdout),
+	)
```

Comment:
Where is this used? I wouldn't expect the default verbosity to be 4 unless we were testing

Reply:
we are using it here https://github.com/openshift/machine-api-operator/pull/1204/commits/e25947807fb043cf3d0ef8d87bcb860cc03ee2a2#diff-1527227bab08db25e1d5effed800dbfa72b143754c3bba342e2b3f9c62cec75cR200 and https://github.com/openshift/machine-api-operator/pull/1204/commits/e25947807fb043cf3d0ef8d87bcb860cc03ee2a2#diff-87fc30a0c4c4f62bdca89d4b09f9b5133e71741dfeb5b2df7260331b698d6698R40

It looks like[klog.New() set a default level of 0](https://github.com/kubernetes/klog/blob/main/klogr/klogr.go#L60-L70).  I'll update the default to be 0 to match.

Reply:
When used in the `main.go`, you need the config to register flags, instead of the `klog.InitFlags` we had used previously IIUC, have you checked that the flags for verbosity are actually working?